lib/trento_web/openapi/schema/cluster.ex

```diff
@@ -138,14 +139,19 @@ defmodule TrentoWeb.OpenApi.Schema.Cluster do
           items: %Schema{type: :string}
         },
         health: ResourceHealth,
-        resources_number: %Schema{type: :integer, description: "Resource number"},
-        hosts_number: %Schema{type: :integer, description: "Hosts number"},
-        cib_last_written: %Schema{type: :string, description: "CIB last written date"},
+        resources_number: %Schema{type: :integer, description: "Resource number", nullable: true},
```

Comment:
why are all of these things nullable now?

Reply:
asnwered in the main comment